packages/theme/README.md

```diff
@@ -125,12 +108,11 @@ const themes = createThemes(
     },
   },
 });
-themes.dark.spacings.medium;
-//=> 20
+console.log(dark.spacing.medium);
```

Comment:
A slight typo here? Overrides for `light`, but prints for `dark`.

Reply:
Good catch, fixed!